sphinx_rtd_theme/layout.html

```diff
@@ -6,10 +6,11 @@
 {%- else %}
   {%- set titlesuffix = "" %}
 {%- endif %}
+{%- set lang_attr = 'en' if language == None else (language | replace('_', '-')) %}
```

Comment:
I wonder if this catches the proper language in most cases. In particular, what happens if `language` is undefined? I believe jinja has a check for this. Have you tried building sets of docs with this patch and different language variables defined?